vue/components/ui/molecules/button-group/button-group.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+    <div class="button-group">
+        <div
+            class="button-container"
+            v-for="item in items"
+            v-bind:key="item.value"
+            v-on:click="onClick($event, item)"
+        >
+            <slot v-bind:item="item" v-bind:selected="valueData === item.value">
+                <button-color
+                    class="button-color"
+                    v-bind:class="buttonClasses(item)"
+                    v-bind:disabled="disabled"

```

Comment:
could be useful if we we're able disable individual buttons 
```suggestion
                    v-bind:disabled="disabled || item.disabled"
```